pkg/runtime/oci/config/root.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+type RuntimeOciRoot interface {
+	GetSpec() *specs.Root
+
+	GetPath() string
+	SetPath(path string)
+
+	GetReadOnly() bool
+	SetReadOnly(enabled bool)
+}
+
+type DefaultRuntimeOciRoot struct {
```

Comment:
Why are each of these `DefaultRuntimeOci*` structs embedding a `RuntimeOciSpec` directly instead of the proper member. For instance, here why are we not having `RuntimeOciSpecRoot *RuntimeOciSpecRoot`?

Reply:
I take the easy way there, because if you embed the proper member, somewhere you have to link it against a common specs.Spec instance to generate JSON at the end

Reply:
Ahhh, makes sense. Thanks for clarifying :)